### DISCUSSION TITLE
How find value edited-second in dataview

### DISCUSSION THREAD

**Discussion by guix54** [DynamicPlayerSector/timethings#15] [General]
hello,
I have this query
```dataview
table edited-seconds as "Durée", file.path
where edited_seconds
sort edited_seconds desc
limit 20
```
but the field edited_seconds is always empty in the result...(but the field in the file not)
I do not find the best way to find in my vault most edited notes :-)

I try to change the frontmatter setting > no effect then disable option

**Comment by plasmabit** [comment]
I like this idea.

I have tried creating a query in my vault, and it works fine. Although it looks slightly different. Hopefully my code will help:

```
TABLE WITHOUT ID
	file.link AS "Note",
	edited_seconds AS "Seconds edited"
WHERE edited_seconds != NULL
SORT edited_seconds DESC
LIMIT 20
```
